test.ts

```diff
@@ -627,3 +627,49 @@ Deno.test("lastInsertedId", function () {
   // will be resetted to 0 again
   assertEquals(db.lastInsertRowId, 0);
 });
+
+Deno.test("outputToObjectArray", function () {
+  const db = new DB();
+
+  const expectedName = "John Doe";
+
+  // Create table and insert value
+  db.query(`CREATE TABLE users (id INTEGER PRIMARY KEY, name VARCHAR(255))`);
```

Comment:
Use `TEXT` instead of `VARCHAR` (see https://www.sqlite.org/datatype3.html#affinity_name_examples)

Reply:
👍